pkg/actors/actors.go

```diff
@@ -65,6 +66,12 @@ type actorsRuntime struct {
 	evaluationChan      chan bool
 }
 
+// ActorCount contain actorType and count of actors each type has
+type ActorCount struct {
+	Type  string `json:"type"`
+	Count int    `json:"count"`
```

Comment:
I would rename this count to ActiveActorsCount. It should be called ActiveActorsCount in the returned response as well.